docs/source/model_design/risk_factors.rst

```diff
@@ -195,7 +195,118 @@ relationships directionally between a cause/exposure and effect/outcome.
+
++----------+---------------------------+---------------------------+
+|          | Drinks coffee             | Does not drink coffee     |
++==========+=============+=============+=============+=============+
+|          | Smoker      | Non-smoker  | Smokers     | Non-smokers |
++----------+-------------+-------------+-------------+-------------+
+| Cases    |      400    |    50       |   200       |     100     |
++----------+-------------+-------------+-------------+-------------+
+| Control  |      100    |    100      |   50        |     200     |
++----------+-------------+-------------+-------------+-------------+
+|Odds ratio| (400/50)÷(100/100) = 8.0  | (200/100)÷(50/200) = 8.0  |
++----------+---------------------------+---------------------------+
+
+The odds of pancreatic cancer are 8 times higher among individuals who smoke than among individuals who do not smoke regardless of whether or not they drink coffee. This then satisfies the first criterion that the factor must be a risk factor for the disease in those unexposed.
+
+Finally, we can be confident that smoking is not on the causal pathway between coffee drinking and pancreatic cancer, so the third criterion for being a confounder is satisfied.
+
+It should now be clearer why smoking acts as a confounder in the association between coffee consumption and pancreatic cancer. Smoking is itself a risk factor for pancreatic cancer, and smoking is also differentially distributed between those who drink and not drink coffee. Thus, when we first looked at the overall association between coffee consumption and pancreatic cancer, the effect of coffee consumption was **mixed up** with the effect of smoking (because there are proportionately more smokers among those who drink coffee than among those who do not). But once we accounted for smoking by looking at the data separately for smokers and non-smokers, it became clear that there is, in fact, no evidence for an association between coffee and pancreatic cancer.
+
+Having established that smoking appears to confound the association between coffee consumption and pancreatic cancer, the final step is to combine, or pool, the data across strata of smoking to obtain a combined, or pooled, estimate of the effect of coffee consumption on pancreatic cancer adjusted for the confounding effect of smoking. This pooled estimate is an average of the effect across all strata of smoking weighted by the size of each stratum. There are statistical methods for obtaining such pooled estimates (e.g. Mantel-Haenszel, regression). There are a number of strategies to minimise or deal with confounding, both when designing a study and in the analysis phase.
+
+.. todo::
+  How does GBD handle confounding
```

Comment:
I think that we covered this a bit in the general causality section, so it might be fine not to add more detail here unless there's anything else specific that you would like to add!

Reply:
Also, it might be helpful to add a section about what we should think about as sim science team members to watch out for confounding. Like when looking for data sources we should check that the studies adjusted for age/sex/other important variables when appropriate.

Reply:
yes exactly!!